### DISCUSSION TITLE
How can I change the decimal separator in FluentNumber field

### DISCUSSION THREAD

**Discussion by lonestarcz** [microsoft/fluentui-blazor#1799] [Q&A]
Hello,

I am working on a project with Czech localization. Even if I have the localization set, the FluentNumberField uses dot as the decimal separator, but it confuses people, because in Czech we use comma as the separator.

How can I set the separator to comma?

I am using .net8 (8.0.3), blazor wasm, fluentUI components 4.6.0

thanks for help

**Comment by vnbaaij** [comment]
Hi, 

Unfortunately, you can't. Tis has been discussed before. Please see #1127 for details.
